integration/suite_test.go

```diff
@@ -0,0 +1,55 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/diskordanz/darksky/config"
+	"github.com/diskordanz/darksky/integration"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+	ctx          context.Context
+	app          *integration.Darksky
+	ts           *httptest.Server
+	mockResponse []byte
+}
+
+func (i *IntegrationTestSuite) SetupTest() {
+	logger := logrus.WithFields(logrus.Fields{
+		"logger": "test",
+	})
+
+	i.app = integration.Init(logger)
+	i.ctx = context.Background()
```

Comment:
you can write here smth like: 

```
i.ctx := context.Background() 
i.ctx = context.WithValue(i.ctx, config.APIKey, "just_some_value") 
```

Reply:
provide key explicitly in suite_test.go?

Reply:
you can use any dummy value instead of the "real" key. Anyway in your test you don't call API directly, you just receive mocked values.. 
Btw, in your realization.. Well, it can be tricky. Let's discuss it face2face at office today, ok?